### ISSUE TITLE
Google preview snippet does not show workspace changes

### ISSUE THREAD

**Issue by Jashi** [Yoast/Yoast-SEO-for-TYPO3#512]
* [x] I've read and understood the [contribution guidelines](https://github.com/Yoast/Yoast-SEO-for-TYPO3/blob/master/.github/CONTRIBUTING.md).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened.
If an editor edits the seo title or seo description in a workspace the changes are shown immediately in the google preview snippet but after saving the changes the preview snippet will go back to showing the live content (because yoast renders the page without workspace mode and "reads" title and description).

### Please describe what you expected to happen and why.
As an editor I expect the preview snippet to show my current code - in or outside of a workspace. Otherwise I have to publish content to see the result which is not desirable.

### How can we reproduce this behavior?
1. Edit the seo title and/or description in workspace
2. Save your changes

### Technical info
* TYPO3 version: 9, 10, 11
* Yoast SEO version:  7.2 and 8.3

**Comment by Jashi** [comment]
Ok, we've used the Hook provided by Yoast to modify the URL for our workspace, so this isn't a problem in our project anymore.

**Comment by RinyVT** [comment]
Hi @Jashi,

I'm planning to take a look at this for the upcoming major version.
Problem with these workspace changes, and for example #319, is that we first do a request to a backend route, which in turn does the actual page request.
The actual page request does not include the current backend user session so it's important that making this a possibility does not introduce security issues.
Was it only a change in the URL for you to make it work, or did you also have to do some other changes?

**Comment by Jashi** [comment]
Yes, we simply changed the URL. We don't have different backend user rights so that was not an issue fortunately.
